sdk/evaluation/azure-ai-evaluation/setup.py

```diff
@@ -72,6 +72,7 @@
         "azure-identity>=1.16.0",
         "azure-core>=1.30.2",
         "nltk>=3.9.1",
+        "azure-ai-inference>=1.0.0b4"
```

Comment:
we can't take a dependency on a non-GA package. For now, can you move this to an extra?